RecoTracker/LSTCore/interface/Common.h

```diff
@@ -35,6 +35,8 @@ namespace lst {
 
   constexpr unsigned int size_superbins = 45000;
 
+  constexpr unsigned int n_triplet_threshold = 1000;
```

Comment:
I think this makes more sense in our other Common.h file, `/interface/alpaka/Common.h` with prefix `HOST_DEVICE_CONSTANT`.